dev/io.openliberty.microprofile.metrics.internal.5.0_fat/fat/src/io/openliberty/microprofile/metrics50/internal/tck/launcher/LibraryRefTest.java

```diff
@@ -72,6 +95,22 @@ public void after() throws Exception {
         }
     }
 
+    /*
+     * Java 11.0.14 known to cause issues.
+     */
+    private static void isJava11014() throws IOException {
+        JavaInfo javaInfo = JavaInfo.forServer(server);
+
+        Log.info(c, "isJava11014",
+                 "Major.minor.micro : [" + JavaInfo.forServer(server).majorVersion()
+                                   + "."
+                                   + JavaInfo.forServer(server).minorVersion()
+                                   + "." + JavaInfo.forServer(server).microVersion()
+                                   + "]");
+        if (javaInfo.majorVersion() == 11 && javaInfo.microVersion() == 17)
```

Comment:
Is the microVersion of 11.0.14 jdk really `17`?

Reply:
No, that was left over :(